src/ddmd/traits.d

```diff
@@ -403,6 +404,50 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         return new IntegerExp(e.loc, false, Type.tbool);
     }
 
+    /**
+     Params: o: the AST-node to check.
```

Comment:
Please read the Ddoc documentation I provided a link for. Note the `o:` in the above line. This is not how it's done and Ddoc will not work with it. It's `o=`.